spec/services/metalv1/oas3.patched/components/schemas/VirtualNetwork.yaml.rej

```diff
@@ -0,0 +1,12 @@
+--- spec/services/metalv1/oas3.patched/components/schemas/VirtualNetwork.yaml
```

Comment:
This patch was adopted upstream since the last sync, so this PR should be updated to remove this `VirtualNetwork.yaml.rej` file and the `spec/services/metalv1/patches/20230801-added-pro-created-at-in-virtual-network.patch` file that was upstreamed.